src/System.IO.Pipelines/ThrowHelper.cs

```diff
@@ -52,6 +52,11 @@ public static void ThrowArgumentOutOfRangeException_BufferRequestTooLarge(int ma
             throw GetArgumentOutOfRangeException_BufferRequestTooLarge(maxSize);
         }
 
+        public static void ThrowObjectDisposedException(string objectName)
+        {
+            throw GetObjectDisposedException(objectName);
+        }
+
```

Comment:
BTW, the reason I had these methods on OwnedBuffer (as opposed on throw helper) is that the ThrowHelper is no public, and so each assembly that implements OwnedBuffer needs its own ThrowHelper. @pakrym.